tests/functional/watcher_controller_test.go

```diff
@@ -20,6 +23,12 @@ var (
 	MinimalWatcherSpec = map[string]interface{}{
 		"databaseInstance": "openstack",
 	}
+
+	MinimalWatcherContainerSpec = map[string]interface{}{
```

Comment:
We are actually not deploying the controller, but I'd also include the `"databaseInstance": "openstack"` in this Spec as it's required for a valid CR.

Reply:
Done!